charts/mageai/Chart.yaml

```diff
@@ -51,11 +51,11 @@ maintainers:
 name: mageai
 dependencies:
   - name: postgresql
-    version: 15.5.15
```

Comment:
this seems a breaking change. I have to roll it back 
<img width="1058" alt="image" src="https://github.com/user-attachments/assets/c364f03d-de6f-49cf-9efc-4476dbfe1eda" />